src/App/App.component.tsx

```diff
@@ -25,6 +25,16 @@ function App() {
             onLoaded={() => console.log('female avatar loaded')}
             fov={45}
             ambientLightIntensity={0}
+            emotion={{
```

Comment:
that part should be deleted after review. Added it to make the reviewer test process easier

Reply:
This example should be added to storybook instead, under the animation example

Reply:
will do